### PR TITLE
Add writeup for November 2022

### PR DESCRIPTION
**Pull request by spikeheap**

### CODE REVIEW

File: _hacks/32.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 ---
 hack_number: 32
 date: 2022-11-26
-upcoming: true
+upcoming: false
```
Comment (lpmi-13) [inline _hacks/32.md:4]: We usually just delete this line, but very nit and not a blocker

File: _hacks/33.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+hack_number: 33
+date: 2023-01-28
+upcoming: true
```
Comment (lpmi-13) [inline _hacks/33.md:4]: I dont actually know what our automation does if this already exists on December 1...but I guess we can find out!

**Review by lpmi-13 — APPROVED** [review]

**Comment by lpmi-13** [comment]
Not a single mention of Cosimo! For shame!!!

**Comment by spikeheap** [comment]
Screenshot 2022-11-28 at 11 43 18
It looks like it's all worked out well

**Comment by lpmi-13** [comment]
well...until our automation tries to add a file on December 1st... :popcorn:

**Pull request merged**